Shared/SmartFeeds/SmartFeed.swift

```diff
@@ -46,7 +46,7 @@ final class SmartFeed: PseudoFeed {
 	}
 	#endif
 
-	private let delegate: SmartFeedDelegate
+	public let delegate: SmartFeedDelegate
```

Comment:
Needed this public (because of `let` it's read-only) so I could conditionally not show the submenu on the TodayFeed context menu. That would have required copying or exposing the private function `todayCutoffDate` on `ArticlesDatabase`.